spec/features/api_access_spec.rb

```diff
@@ -14,13 +14,31 @@ def response_json
   end
 
   describe "api json export" do
-    it 'routes to the all action on default' do
+    it 'exports all organizations as json by default' do
       create(:organization, name: "organization foo")
       create(:organization)
       sign_in_as_admin
-      get "/api"
+      get "/api.json"
       expect(response_json.size).to eq(2)
       expect(response_json[0]['name']).to eq("organization foo")
     end
   end
+
+  describe "api markdown export" do
+    it 'spec_name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.